### PR TITLE
docs(ext-plugin): init docs

### PR DESCRIPTION
**Pull request by spacewander**
Signed-off-by: spacewander <spacewanderlzx@gmail.com>

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/plugins/ext-plugin-pre-req.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+To know what is the plugin runner, see [external plugin](../external-plugin.md) section.
+
+The result of external plugins execution will affect the behavior of the current request.
+
+## Attributes
+
+| Name      | Type          | Requirement | Default    | Valid                                                                    | Description                                                                                                                                         |
+| --------- | ------------- | ----------- | ---------- | ------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| conf     | array        | optional    |              | [{"name": "ext-plugin-A", "value": "{\"enable\":\"feature\"}"}] |     The plugins list which will be executed at the plugin runner and their configuration    |
```
Comment (nic-chen) [inline docs/en/latest/plugins/ext-plugin-pre-req.md:45]: with their configurations?

File: docs/en/latest/plugins/ext-plugin-pre-req.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+```shell
+curl -i http://127.0.0.1:9080/index.html
+```
+
+You will see the configured plugin runner will be hit and plugin `ext-plugin-A`
+is executed at that side.
+
+## Disable Plugin
+
+When you want to disable the serverless plugin, it is very simple,
```
Comment (nic-chen) [inline docs/en/latest/plugins/ext-plugin-pre-req.md:83]: serverless —> external

**Review by nic-chen — APPROVED** [review]

File: docs/en/latest/external-plugin.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
+---
+title: external plugin
```
Comment (juzhiyuan) [inline docs/en/latest/external-plugin.md:2]:
```suggestion
title: External Plugin
```

File: docs/en/latest/external-plugin.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+## Supported plugin runners
+
+Java: https://github.com/apache/apisix-java-plugin-runner
+Go: https://github.com/apache/apisix-go-plugin-runner
+
+## Configuration for plugin runner in APISIX
+
+To run plugin runner in the prod, add the section below to `config.yaml`:
+
+```
```
Comment (juzhiyuan) [inline docs/en/latest/external-plugin.md:59]:
```suggestion
```yaml
```

**Review by juzhiyuan — APPROVED** [review]

File: docs/en/latest/external-plugin.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+When you configure a plugin runner in APISIX, APISIX will run the plugin runner
+as a subprocess. The process will belong to the same user of the APISIX
+process. When we restart or reload APISIX, the plugin runner will be restarted too.
+
+Once you have configured `ext-plugin-*` plugins for a given route, the requests
+which hit the route will trigger RPC call from APISIX to the plugin runner via
+unix socket.
+
+The plugin runner will handle the PRC call, create a fake request at its side,
```
Comment (tokers) [inline docs/en/latest/external-plugin.md:44]:
```suggestion
The plugin runner will handle the RPC call, create a fake request at its side,
```

**Review by tokers — APPROVED** [review]

**Pull request merged**